rpa-manager/modules/ROOT/pages/botmanagement-troubleshoot.adoc

```diff
@@ -76,6 +78,24 @@ An IT administrator can configure the content of the package in the RPA Bot Conf
 . Click *Download Logfiles* (image:download-icon.png[disk-with-downwards-arrow symbol,1.5%,1.5%]) in the row of the bot you want to read the log files from.
 . Extract the contents and view the relevant files.
 
+== Turn On Debug Logging
+
+Debug Logging is supported by RPA Bots with version 1.3.3 and later.
+
+If a process execution fails when a bot runs correctly, turn on debug logging for at least the time span of the next process run to get help detecting the cause.
+
+Then download the log files of the bot and send them to the support team if you need help analyzing the cause of the failure.
+
+To turn on debug logging, complete the following steps:
+
+. Open the *Bots* view of the *Bot Management* module.
+. On the panel of the bot to debug, click *Turn on debug logging* (image:debug-icon.png[bug symbol,1.5%,1.5%]).
+. In the *Enable time limited debug logging* window, enter a time interval in minutes and click *Start debug logging*.
```

Comment:
You can simply say:
"In *Enable time limited debug logging*, enter a time interval in minutes and click *Start debug logging*."

Reply:
This would need to be changed everywhere to be consistent. I'll put it in the backlog....